### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by dlikhten** [cantierecreativo/redux-bees#74]
Looks like it has been a while since there was a commit to this project. Is it dead? Actively looking for other maintainers? I noticed a bunch of pull requests open too. Just wanted to get a sense of where things are.

**Comment by YuriScarbaci** [comment]
Can't say anything official, but you are right latest commit was 2 months ago and at best this looks scarcely maintained,

We are using this in two of my projects and it's `stable` enough to develop a fully working project, even if sometimes it get a little bit tricky, and we aren't using any decorator ( we implemented some hooks that dispatch redux-bee requests and suchs helpers, also to manage caching on a busisness level logic basis)

This fall shorts when your projects requires a bigger degree of customization, and the biggest flaw I detected so far is that `DELETE` won't accept a body in the payload...

https://github.com/cantierecreativo/redux-bees/issues/63
https://github.com/cantierecreativo/redux-bees/issues/68

this project surely deserves to be kept alive in my opinion

**Comment by dlikhten** [comment]
Right. I was considering forking. I've already made some customization in my repository and am thinking of making that public. For example the reducer doesn't support optional relationships.

**Comment by Lorezz** [comment]
Hi, if anyone is interested in becoming a maintainer of this repository, they can write about it here.  Thanks

**Comment by pompitasarkar** [comment]
Hi,

We've been actively using this module in our project, but we're facing an impasse with 401, and 204 Delete which doesn't accept body. We wanted to make modifications to the same, so we've forked your repo, but it's becoming impossible to build. npm install throws babel errors. Do you have any .babelrc file which would help us build the project? 

That would be a life saver. Thanks.

**Comment by joeyates** [comment]
@pompitasarkar I'm afraid that offhand we can't help you with .babelrc changes. Please open an issue with the error output and steps to reproduce.
